src/main/resources/db/migration/V1_1112__CASELOAD_CURRENT_ACCOUNTS_TXNS.sql

```diff
@@ -58,3 +58,5 @@ CREATE UNIQUE INDEX CASELOAD_CURRENT_ACCT_TXNS_PK ON CASELOAD_CURRENT_ACCOUNTS_T
 CREATE INDEX CSLD_CAT_AC_PRD_F2 ON CASELOAD_CURRENT_ACCOUNTS_TXNS (ACCOUNT_PERIOD_ID);
 CREATE INDEX CASELOAD_CURRENT_ACTS_TXNS_NI1 ON CASELOAD_CURRENT_ACCOUNTS_TXNS (CASELOAD_ID, ACCOUNT_CODE, ACCOUNT_PERIOD_ID);
 CREATE INDEX CSLD_CAT_AC_CODE_F1 ON CASELOAD_CURRENT_ACCOUNTS_TXNS (ACCOUNT_CODE);
+
+create sequence CASELOAD_CURRENT_ACCOUNT_ID start with 1;
```

Comment:
Not sure if there's a precedence but I thought we put these into v1_0_SEQUENCES ?